vimeo-networking/src/main/java/com/vimeo/networking/model/error/ErrorCode.java

```diff
@@ -196,13 +205,99 @@
     INVALID_INPUT_BAD_CLIP_PRIVACY_EMBED,
     @SerializedName("2240")
     INVALID_INPUT_BAD_CLIP_PRIVACY_COMMENTS,
-
+    @SerializedName("2241")
+    INVALID_INPUT_BAD_USER_URI,
+    @SerializedName("2242")
+    INVALID_INPUT_NO_USER_URI,
+    @SerializedName("2244")
+    INVALID_INPUT_NO_CLIP_USERS,
+    @SerializedName("2245")
+    INVALID_INPUT_EMPTY_USERS_ARRAY,
+    @SerializedName("2246")
+    CLIP_PRIVACY_NOT_SET_TO_USERS,
+    @SerializedName("2247")
+    INVALID_INPUT_NO_CLIP_PRIVACY_WHEN_SETTING_USERS,
+    @SerializedName("2248")
+    INVALID_INPUT_BAD_CLIP_PRIVACY_FOR_SETTING_USERS,
+    @SerializedName("2249")
+    INVALID_INPUT_BAD_CLIP_DESCRIPTION_TYPE,
+    @SerializedName("2250")
+    INVALID_INPUT_CLIP_NAME_TOO_LONG,
+    @SerializedName("2251")
+    INVALID_INPUT_CLIP_DESCRIPTION_TOO_LONG,
+    @SerializedName("2252")
+    INVALID_INPUT_BAD_CLIP_NAME_TYPE,
+    @SerializedName("2253")
+    INVALID_INPUT_EMPTY_USER_NAME,
+    @SerializedName("2409")
+    USER_NOT_ALLOWED_TO_SET_PUBLIC_OR_NOBODY_CLIP_PRIVACY,
+    @SerializedName("2410")
+    USER_NOT_ALLOWED_TO_SET_USERS_CLIP_PRIVACY,
+    @SerializedName("2411")
+    USER_NOT_ALLOWED_TO_SET_DISABLE_CLIP_PRIVACY,
+    @SerializedName("2412")
+    USER_NOT_ALLOWED_TO_SET_CONTACTS_CLIP_PRIVACY,
+    // Upload
     @SerializedName("8002")
     UNABLE_TO_UPLOAD_VIDEO_MISSING_USER_ID_FOR_AUTHENTICATION_TOKEN,
     @SerializedName("3400")
     USER_NOT_ALLOWED_TO_UPLOAD_VIDEO_UNVERIFIED_EMAIL,
-    @SerializedName("3401")
-    USER_NOT_ALLOWED_TO_UPGRADE_TO_1080_VIDEO,
+    @SerializedName("3401") @Deprecated USER_NOT_ALLOWED_TO_UPGRADE_TO_1080_VIDEO,
```

Comment:
Since I believe we are the only ones consuming this library at the moment, I am in favor of just deleting this error code, rather than deprecating it. I recently had a similar issue, and opted for deletion.